frontend/packages/core/src/Input/checkbox.tsx

```diff
@@ -1,4 +1,6 @@
 import React from "react";
```

Comment:
notice the name prop doesn't appear in storybook unless the import is like this

```suggestion
import * as React from "react";
```